src/components/DetailsMenu/DetailsMenuCard/index.tsx

```diff
@@ -17,41 +19,56 @@ interface DetailsMenuProps {
 }
 
 const DetailsMenuCard = forwardRef<HTMLDivElement, DetailsMenuProps>(
-  ({ "data-cy": dataCy }, ref) => (
-    <Container ref={ref} data-cy={dataCy}>
-      <Row>
-        <Column>
-          <SearchRangeInput />
-          <WrapToggle />
-          <CaseSensitiveToggle />
-        </Column>
-        <Column>
-          <FilterLogicToggle />
-          <ExpandableRowsToggle />
-          <PrettyPrintToggle />
-        </Column>
-      </Row>
-      <ButtonRow />
-      <CLIInstructions />
-    </Container>
-  )
+  ({ "data-cy": dataCy }, ref) => {
+    const [selectedTab, setSelectedTab] = useState(0);
+    return (
+      <Container ref={ref} data-cy={dataCy}>
+        <H3>Parsley Settings</H3>
+        <Tabs
+          aria-label="Details Card Tabs"
+          selected={selectedTab}
+          setSelected={setSelectedTab}
+        >
+          <Tab name="Log Structure">
```

Comment:
What do you think of naming this something like "Search & Filter" or "Search & Filter Config"? That seems like the overarching theme

Reply:
How about Search & Filter Preferences to keep verbiage between the tabs consistent?

Reply:
![image](https://github.com/evergreen-ci/parsley/assets/4605522/1372764b-7e22-428f-a2cf-358fd2c0729a)

Reply:
Works for me! Or just "Search & Filter" and "Log Viewing" since the menu title already says "Settings" :)

Reply:
LGTM cc @willperez23